docs/extproc-config.yaml

```diff
@@ -0,0 +1,32 @@
+llmRequestCosts:
+- metadataKey: used_token
+  type: InputToken
+metadataNamespace: ai_gateway_llm_ns
+modelNameHeaderKey: x-model-name
+rules:
+- backends:
+  - auth:
+      apiKey:
+        filename: /tmp/TestWithRealProviders3286901600/004/open-ai-api-key
```

Comment:
i already see this won't work on anyone's machine, which is why I insisted on everything being tested.